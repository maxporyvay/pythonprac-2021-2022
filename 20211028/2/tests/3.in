print(*list(slide([[1],2,None,'nlkjlj'], 4)))