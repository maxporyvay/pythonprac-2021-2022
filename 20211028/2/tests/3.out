[1] 2 None nlkjlj 2 None nlkjlj None nlkjlj nlkjlj
